src/resolve-celo.ts

```diff
@@ -0,0 +1,76 @@
+import Web3 from 'web3'
+import { Address } from '@celo/base/lib/address'
+import { AbiItem } from 'web3-utils'
+import { Contract } from 'web3-eth-contract'
+import { OdisUtils } from '@celo/identity'
+import { AuthSigner, ServiceContext } from '@celo/identity/lib/odis/query'
+
+import { NameResolver, NameResolutionResults, ResolutionKind } from './types'
+
+export class ResolveCelo implements NameResolver {
+  private federatedAttestationsContract: Contract
+  private authSigner: AuthSigner
+  private account: Address
+  private serviceContext: ServiceContext
+
+  static readonly FederatedAttestationsContractAbi: AbiItem[] =
+    require('./abis/FederatedAttestations.json').abi
+  static readonly AlfajoresFederatedAttestationsProxyContractAddress: Address =
+    '0x70F9314aF173c246669cFb0EEe79F9Cfd9C34ee3'
+
+  constructor({
+    providerUrl,
+    federatedAttestationsProxyContractAddress,
+    authSigner,
+    account,
+    serviceContext,
+  }: {
+    providerUrl: string
+    federatedAttestationsProxyContractAddress: Address
+    authSigner: AuthSigner
```

Comment:
I'd like to avoid exposing types of `AuthSinger` and `ServiceContext`.

In general, it seems prudent to avoid exposing specific implementation details of how we might resolve things.  E.g., it shouldn't matter to the caller of this library if we use web3.js or ethers; or the ODIS SDK or something else; etc.

We could check with the identity team if we can avoid exposing this.  E.g., is it OK with them if we manually construct a ServiceContext from constants vs using the SDK helpers?

Reply:
Totally fine to declare your own `ServiceContext` constant, but using the getter in the SDK ensures your info is the most up to date.

Reply:
I'm a little confused around your point about exposing how things are resolved. You can use the ODIS SDK with web3js or ethers already, and you can even forgo the SDK and make queries directly to ODIS if you really wanted to

Reply:
I don't want to expose the implementation details of resolve-kit to the caller.  E.g., if resolve-kit uses web3js, but then wants to change to ethers, the users of this library shouldn't notice.

re `ServiceContext`: that's kinda my question.  Do you want to manage the specific constants via the SDK?  or are you OK if users (i.e., Valora) use them directly?  I think those two options have different maintenance implications on the ODIS SDK.

Reply:
@silasbw if I'm understanding this correctly, you don't want the caller of this library to need to pass in arguments of types ServiceContext and AuthSigner since those types are defined within the identity sdk, right? 
I think it should work fine to just pass in a private key (instead of AuthSigner) and environment name (i.e. 'mainnet' or 'alfajores') and construct the ServiceContext and AuthSigner to pass in to the identity sdk within the ResolveCelo constuctor.
Does that address your concern?

Reply:
Kinda.  We _try_ (with varying degrees of success) to avoid interfaces that take a specific network name, and instead try to use the values directly that are connoted by that network name (e.g., ~alfajores~ -> https://alfajores-forno.celo-testnet.org; or ~local~ -> http://localhost:7545).  We hope this facilitates more controlled integration tests, but we make compromises all the time and it sounds non-trival to spin up an ODIS instance, so what you suggest is fine.

Reply:
did you decide to keep these parameters as type ServiceContext and AuthSigner?

Reply:
Yes, for now :/, if that's OK.